programs/util.c

```diff
@@ -54,14 +54,24 @@ int UTIL_getFileStat(const char* infilename, stat_t *statbuf)
 int UTIL_setFileStat(const char *filename, stat_t *statbuf)
 {
     int res = 0;
+#if defined(_WIN32) || (PLATFORM_POSIX_VERSION < 200809L)
     struct utimbuf timebuf;
+#else
+    struct timespec timebuf[2];
+#endif
 
     if (!UTIL_isRegularFile(filename))
         return -1;
 
+#if defined(_WIN32) || (PLATFORM_POSIX_VERSION < 200809L)
     timebuf.actime = time(NULL);
     timebuf.modtime = statbuf->st_mtime;
     res += utime(filename, &timebuf);  /* set access and modification times */
+#else
+    timebuf[0].tv_sec = time(NULL);
```

Comment:
Also, can you initialize the `tv_nsec` fields to 0?

Reply:
Initialized.